sdk/tests/conformance/reading/00_test_list.txt

```diff
@@ -1,3 +1,3 @@
 read-pixels-pack-alignment.html
 read-pixels-test.html
-
+fbo-remains-unchanged-after-read-pixels.html
```

Comment:
@zhenyao , why need to add 1.0.4 for the new test? Is the reason that it is CMAA special case.